reference/json-rpc/filecoin.md

```diff
@@ -2,14 +2,41 @@
 
 ## FilecoinAddressToEthAddress
 
-FilecoinAddressToEthAddress converts an f410 or f0 Filecoin Address to an EthAddress
+FilecoinAddressToEthAddress converts any Filecoin address to an EthAddress.
+
+This method supports all Filecoin address types:
+
+- "f0" and "f4" addresses: Converted directly.
+- "f1", "f2", and "f3" addresses: First converted to their corresponding "f0" ID address, then to an EthAddress.
+
+Requirements:
+
+- For "f1", "f2", and "f3" addresses, they must be instantiated on-chain, as "f0" ID addresses are only assigned to actors when they are created on-chain.
+  The simplest way to instantiate an address on chain is to send a transaction to the address.
+
+Note on chain reorganizations:
```

Comment:
Let's move the Note section above Requirements. It will be nice to use the Note style for Markdown format.